docs/src/examples/components/Ref/Types/RefExampleForwardRef.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Grid, Ref, Segment } from '@stardust-ui/react'
+
+const ExampleButton = React.forwardRef<HTMLButtonElement>((props, ref) => (
```

Comment:
The usage of generic in docs :( It's impossible to omit it there.

Reply:
yes, and this is definitely not the problem of this PR :) seems that we should use JS engine to process this code, as docs are aimed to present **JS** variant of examples to the client (this is the case for now, at least)

Reply:
I'm planning to left this thing as is and push https://github.com/stardust-ui/react/issues/495 that will resolve all our issues.